libs/cli/langgraph_cli/config.py

```diff
@@ -840,6 +850,15 @@ class LocalDeps(NamedTuple):
         additional_contexts: A list of paths to directories that contain local
             dependencies in parent directories. These directories are added to the
             Docker build context to ensure that the Dockerfile can access them.
+
+        pkgs_missing_reqs: A list of packages that need requirements.txt generated.
```

Comment:
I think for these descriptions, ir makes more sense to put them inline with the ReqGenSpec object above in this case (since we do have a typed object for it)